### PR TITLE
Update Demand Media related rulesets

### PR DESCRIPTION
**Pull request by cschanaj**
### List related PRs if any

- #17965

### CODE REVIEW

File: src/chrome/content/rules/GolfLink.com.xml [hunk 1]
```diff
@@ -1,34 +1,13 @@
...
+		- blog.golflink.com
 -->
 <ruleset name="GolfLink.com (partial)">
-
-	<target host="i.glimg.net" />
-	<target host="ui.glimg.net" />
 	<target host="golflink.com" />
-	<target host="cdn-www.golflink.com" />
-	<target host="golflink.net" />
-	<target host="www.golflink.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/GolfLink.com.xml]: Why did you remove all other targets?

Comment (cschanaj) [reply src/chrome/content/rules/GolfLink.com.xml]: I missed some targets when I was trying to extract them from the ruleset. I pushed more changes to includes more targets. Thanks.

File: src/chrome/content/rules/GolfLink.net.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		SSL peer certificate was not OK:
+		- golflink.net
+		- www.golflink.net
+		- staging.golflink.net
+		- a.staging.golflink.net
+		- b.staging.golflink.net
+-->
+<ruleset name="Golflink.net" default_off="cert-invalid">
+	<target host="golflink.net" />
+	<target host="www.golflink.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/GolfLink.net.xml]: `cdn-www.golflink.com`
